docs/docs/faq.md

```diff
@@ -2,8 +2,17 @@
 
 ## Data Collection Client
 
-#### Is there a way to capture more than one count at a time?
-The Suma data collection client has a hidden 'multiCount' feature that enables the capture of multiple counts with the same activities. To access this feature, append the query string `?multiCount=true` to the URL of your instance of the Suma data collection client, e.g. YOUR_SUMA_CLIENT_URL/?multiCount=true. Please note that all counts will have the same timestamp. **We recommended that this feature be used cautiously, as it is much easier to enter large amounts of data this way. Use at your own risk!**
+#### Is there a more permanent way to enable settings.
+Settings by themselves are pretty sticky, they only disappear when history is deleted. With that in mind you can make settings more sticky by adding them to your url as a query string. `variable=setting.` There is no limit on 
```

Comment:
Looks like part of a sentence was left off here.  Is this what you were going to say?
```suggestion
Settings by themselves are pretty sticky, they only disappear when history is deleted. With that in mind you can make settings more sticky by adding them to your url as a query string. `variable=setting.` There is no limit on the number of settings added to the query string.
```